download-links.html

```diff
@@ -18,23 +18,23 @@ <h2>Source Downloads</h2>
 <h2>Linux Downloads</h2>
 
 <ul>
-<li>32 bit x86: <a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">https://download.cuberite.org/linux-i386/Cuberite.tar.gz</a></li>
-<li>64 bit x86: <a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz</a></li>
+<li>x86: <a href="https://download.cuberite.org/linux-i386/Cuberite.tar.gz">https://download.cuberite.org/linux-i386/Cuberite.tar.gz</a></li>
+<li>x86-64: <a href="https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz">https://download.cuberite.org/linux-x86_64/Cuberite.tar.gz</a></li>
 <li>armhf (Raspberry Pi): <a href="https://download.cuberite.org/linux-armhf-raspbian/Cuberite.tar.gz">https://download.cuberite.org/linux-armhf-raspbian/Cuberite.tar.gz</a></li>
 <li>aarch64: <a href="https://download.cuberite.org/linux-aarch64/Cuberite.tar.gz">https://download.cuberite.org/linux-aarch64/Cuberite.tar.gz</a></li>
 </ul>
 
 <h2>Windows Downloads</h2>
 
 <ul>
-<li>32 bit x86: <a href="https://download.cuberite.org/windows-i386/Cuberite.zip">https://download.cuberite.org/windows-i386/Cuberite.zip</a></li>
-<li>64 bit x86: <a href="https://download.cuberite.org/windows-x86_64/Cuberite.zip">https://download.cuberite.org/windows-x86_64/Cuberite.zip</a></li>
+<li>x86: <a href="https://download.cuberite.org/windows-i386/Cuberite.zip">https://download.cuberite.org/windows-i386/Cuberite.zip</a></li>
+<li>x86-64: <a href="https://download.cuberite.org/windows-x86_64/Cuberite.zip">https://download.cuberite.org/windows-x86_64/Cuberite.zip</a></li>
 </ul>
 
 <h2>macOS Downloads</h2>
 
 <ul>
-<li>64 bit x86: <a href="https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz">https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz</a></li>
+<li>x86-64: <a href="https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz">https://download.cuberite.org/darwin-x86_64/Cuberite.tar.gz</a></li>
```

Comment:
What did I say in the last PR. IIAB, DFI! I can't see any benefit to changing these: they are perfectly legible for all our users, and the proposed replacements are not any more consistent (in fact, less so than before!).